Failed Test: test_history.py::test_run_with_None

Stack Trace:
```
______________________________ test_run_with_None ______________________________

mock_telebot = <MagicMock name='telebot' id='139784580169232'>
mocker = <pytest_mock.plugin.MockerFixture object at 0x7f2222362550>

    @patch("telebot.telebot")
    def test_run_with_None(mock_telebot, mocker):
        mocker.patch.object(history, "helper")
        history.helper.getUserHistory.return_value = None
        print("Is it None?", history.helper.getUserHistory.return_value)
        MOCK_Message_data = create_message("Hello")
        mc = mock_telebot.return_value
        mc.reply_to.return_value = True
        history.run(MOCK_Message_data, mc)
>       assert mc.reply_to.called
E       AssertionError: assert False
E        +  where False = <MagicMock name='telebot().reply_to' id='139784577704592'>.called
E        +    where <MagicMock name='telebot().reply_to' id='139784577704592'> = <MagicMock name='telebot()' id='139784577667792'>.reply_to

test/test_history.py:89: AssertionError
```

--- a/test/test_history.py
+++ b/test/test_history.py
@@ -86,4 +86,4 @@ def test_run_with_None(mock_telebot, mocker):
     mc = mock_telebot.return_value
     mc.reply_to.return_value = True
     history.run(MOCK_Message_data, mc)
-    assert mc.reply_to.called
+    assert mc.send_message.called

